Uplift/Views/Cells/GymWeekCell.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  GymWeekCell.swift
+//  Uplift
+//
+//  Created by Phillip OReggio on 10/6/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+class GymWeekCell: UICollectionViewCell {
+
+    // MARK: - Identifier
+    static let identifier = Identifiers.gymWeekCell
+
+    // MARK: - Views
+    private var dayLabel: UILabel!
+    private var backCircle: UIView!
+
+    // MARK: - Info
+    /// Weekday this cell represents
+    private var day = WeekDay.sunday
+    /// Weekday representing current day of week
+    private var today = WeekDay.sunday
```

Comment:
we should make these optional and keep them as `nil` initially instead of giving them a default value of `sunday`